README.md

```diff
@@ -119,49 +119,111 @@ bootstrap VM and master nodes using the Terraform providers for libvirt
 
 You can also see the status of the bootkube.sh script which is running via
 `journalctl -b -f -u bootkube.service`.
 
 ## Interacting with the deployed cluster
 
+Consider `export KUBECONFIG=<path-to-config>` to avoid using the `--kubeconig` flag on each command.
+
 When the master nodes are up and the cluster is active, you can interact with the API:
 
 ```
-$ oc --config ocp/${CLUSTER_NAME}/auth/kubeconfig get nodes
+$ oc --kubeconfig ocp/${CLUSTER_NAME}/auth/kubeconfig get nodes
 NAME       STATUS    ROLES     AGE       VERSION
 master-0   Ready     master    20m       v1.12.4+50c2f2340a
 master-1   Ready     master    20m       v1.12.4+50c2f2340a
 master-2   Ready     master    20m       v1.12.4+50c2f2340a
 ```
 
+### GUI
+
 Alternatively it is possible to manage the cluster using OpenShift Console web UI.
 The URL can be retrieved using
 ```
 oc get routes --all-namespaces | grep console
 ```
 By default, the URL is https://console-openshift-console.apps.ostest.test.metalkube.org
 
-Accessing Console running on virtualized cluster from local web browser requires additional
-setup on local machine:
+Accessing the web Console running on virtualized cluster from local web browser requires additional setup on local machine and the virt host to enable forwarding to cluster's VMs.
 
-Add entry to ```/etc/hosts```:
-```
-# /etc/hosts
-192.168.111.4   console-openshift-console.apps.ostest.test.metalkube.org console openshift-authentication-openshift-authentication.apps.ostest.test.metalkube.org api.ostest.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.ostest.test.metalkube.org alertmanager-main-openshift-monitoring.apps.ostest.test.metalkube.org kubevirt-web-ui.apps.ostest.test.metalkube.org oauth-openshift.apps.ostest.test.metalkube.org grafana-openshift-monitoring.apps.ostest.test.metalkube.org
-```
-Run sshuttle:
+There are two ways to achieve this, by using `sshutle` or `xinetd`.
 
-```
-sshuttle -r <user>@<virthost> 192.168.111.0/24
-```
+### sshutle (works only with IPv4)
+
+1. On your local machine install `sshutle`
+   
+2. Add entry to `/etc/hosts`
+    
+    ```
+    192.168.111.4 console-openshift-console.apps.ostest.test.metalkube.org console openshift-authentication-openshift-authentication.apps.ostest.test.metalkube.org api.ostest.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.ostest.test.metalkube.org alertmanager-main-openshift-monitoring.apps.ostest.test.metalkube.org kubevirt-web-ui.apps.ostest.test.metalkube.org oauth-openshift.apps.ostest.test.metalkube.org grafana-openshift-monitoring.apps.ostest.test.metalkube.org
+    ```
+
+3. Run sshuttle on the local machine
+
+    ```
+    sshuttle -r <user>@<virthost> 192.168.111.0/24
+    ```
+
+### xinetd
+
+This approach uses xinetd instead of iptables to allow IPv4 to IPv6 forwarding.
+
+1. Install xinetd
+
+    ```
+    sudo yum install xinetd -y
+    ```
+
+2. Create the xinetd config file
+
+    ```
+    mkdir -p /etc/xinetd.d/
```

Comment:
I did not require this step

Reply:
Thanks, I'll remove this step.
Also the dir vs file comment was misleading.